### ISSUE TITLE
Clean and Fast indication for dependency versioning

### ISSUE THREAD

**Issue by drkhodakarami** [emilyploszaj/trinkets#314]
When I read the readme on the first page of repo, there is a guide for adding dependency for mod developers, how ever, there is no idication of where we can lookup and find proper version for each minecraft version. Many github pages have little gadget that show the latest version!
Also, I came to the repo from curseforge and this makes things more confusing. If the project is on curseforge, why not using it's maven for dependency?! If there is a special case and we need to follow the way from the readme on git hub, please make it easy to find the versioning per minecraft version for dependency and put the guide both in readme and the quick start tutorial on wiki.

**Comment by ByteSizeDev** [comment]
Yes I am having the same issue, perhaps you can help. I do not know what do put in these brackets in the build.gradle file ${trinkets_version} under dependencies. I'm trying to work on 1.19.4 MC
